.buildkite/scripts/standup-gcp-cluster.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Parse command line arguments
+while [[ "$#" -gt 0 ]]; do
+    case $1 in
+        --tf_dir) TF_DIR="$2"; shift ;;
+        --prefix) PREFIX="$2"; shift ;;
+        --gcp_creds) GCP_CREDS="$2"; shift ;;
+        --cluster_type) TASK_NAME="$2"; shift ;;
+        --image) IMAGE="$2"; shift ;;
+        *) echo "Unknown parameter passed: $1"; exit 1 ;;
+    esac
+    shift
+done
+
+# Check if TF_DIR and PREFIX are set
+if [ -z "$TF_DIR" ] || [ -z "$PREFIX" ] || [ -z "$GCP_CREDS" ] || [ -z "$TASK_NAME" ]; then
+    echo "TF_DIR : $TF_DIR"
+    echo "TASK_NAME : $TASK_NAME"
+    echo "TF_DIR, PREFIX, CLUSTER_TYPE and GCP_CREDS must be set. Exiting."
+    exit 1
+fi
+
+cd "$TF_DIR" || exit 1
+export HOSTS_FILE_DIR="$(pwd)/../../artifacts/hosts_gcp_${PREFIX}.ini"
+export KEY_FILE="$(pwd)/../../artifacts/testkey"
+if [ "$TF_DIR" == "gcp" ]; then
+  export HOSTS_FILE_DIR="$(pwd)/../artifacts/hosts_gcp_${PREFIX}.ini"
+  export KEY_FILE="$(pwd)/../artifacts/testkey"
+fi
+
+ssh-keygen -t rsa -b 4096 -C "test@redpanda.com" -N "" -f "$KEY_FILE" <<< y && chmod 0700 "$KEY_FILE"
+
+# Trap to handle terraform destroy on exit
+trap cleanup EXIT INT TERM
+cleanup() {
+    error_code=$?
+    terraform destroy --auto-approve --var="gcp_creds=$GCP_CREDS" --var="deployment_prefix=$PREFIX" --var="public_key_path=$KEY_FILE" --var="project_name=hallowed-ray-376320" --var="hosts_file=$HOSTS_FILE_DIR"
```

Comment:
Does fixing a project name prevent running 2 CI jobs at the same time?

Reply:
No, because we use separate deployment ids